provision/vvv-init.sh

```diff
@@ -54,6 +55,31 @@ fi
 cp -f "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf.tmpl" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 sed -i "s#{{DOMAINS_HERE}}#${DOMAINS}#" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 
+
+# Proxy images from live site
+if [[ "${LIVE_URL}" != http* ]]; then
+	LIVE_URL=''
```

Comment:
If a user enters `example.com` this will get triggered but there's no explanation